dll/win32/kernel32/kernel32.spec

```diff
@@ -593,7 +593,7 @@
 @ stub -version=0x600+ GetSystemDEPPolicy
 @ stdcall GetSystemDefaultLCID()
 @ stdcall GetSystemDefaultLangID()
-@ stub -version=0x600+ GetSystemDefaultLocaleName
+@ stdcall -stub -version=0x600+ GetSystemDefaultLocaleName(wstr long)
```

Comment:
I'd argue the first parameter is a `ptr` since it won't be valid on input

Reply:
Fixed